include/export.bash

```diff
@@ -0,0 +1,122 @@
+export-logs() {
+    declare desc="Exports and anonymizes logs for fault analysis. Usage: cbd export-logs [archive_name]"
+    declare archivename=$1
+    if [[ -z $archivename ]] || [[ "/" == $archivename ]]; then
+        archivename="cbd_export_$(date +%s)"
+        warn "Invalid archive filename. Generated filename: $archivename"
+    fi
+
+    local collectdir="collected_tmp"
+    mkdir -p $collectdir 
+
+    export-psql $collectdir
+    export-prerequisites $collectdir
+
+    export-cbd-env-show $collectdir
+    export-cbd-doctor $collectdir
+    export-cbd-version $collectdir
+    export-cbd-ps $collectdir
+
+    anonymize-exported-files $archivename
+
+    archive-directory $archivename $collectdir
+}
+
+export-psql() {
+    local psqlcmd="docker exec -t cbreak_commondb_1 cat /var/lib/postgresql/.psql_history"
```

Comment:
I prefer to use docker run command with alpine container

Reply:
I cannot use `docker run` here, because the container is already running.

Reply:
What is the benefit for running and new container instead of a simple exec command?

Reply:
Because it works if database is not running somehow